apps/admin_app/lib/admin_app_web/views/general_settings_view.ex

```diff
@@ -3,6 +3,8 @@ defmodule AdminAppWeb.GeneralSettingsView do
   alias Snitch.Data.Model.ProductBrand
   alias Snitch.Data.Model.GeneralConfiguration, as: GCModel
 
+  @currencies ["USD", "INR", "GDP", "EUR"]
```

Comment:
This has to come from core only, we are voilating [DRY](https://metova.com/dry-programming-practices/) here.